tests/testutils.py

```diff
@@ -218,3 +218,49 @@ def send(self, request, **kwargs): # pylint: disable=arguments-differ
                 resp.raw = io.BytesIO(response.encode())
                 break
         return resp
+
+def build_mock_condition(name, condition):
+    return {
+        'name': name,
+        'condition': condition,
+        # ... other relevant fields ...
```

Comment:
Nit: unless we're planning to add other fields, I'm not seeing much value in saying other fields could be added

Reply:
done, recent commit : link[](https://github.com/firebase/firebase-admin-python/pull/824/files#diff-9d24dc3dc9330ef153577f547b4024b50a095ce91290796e44b7a59dcd7b9119)